charts/prometheus-federator/2.0.0+up0.2.1/Chart.yaml

```diff
@@ -8,6 +8,7 @@ annotations:
   catalog.cattle.io/provides-gvr: helm.cattle.io.projecthelmchart/v1alpha1
   catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
   catalog.cattle.io/release-name: prometheus-federator
+  catalog.cattle.io/hidden: "true"
```

Comment:
My knowledge of the chart repo might be out of date, Idon't think this will work because the tgz file of it and the entry in the index.yaml do not have the change.

Reply:
Yes thats why the validation failed. We did make zip and committed the changes to the index and tar

Reply:
I forgot to commit after running the make zip command